Snakefile

```diff
@@ -56,6 +58,17 @@ def devtools_install():
         devtools_cmd = f"devtools::install_github('lculibrk/Ploidetect', ref = '{ver}')"
     return f'"{devtools_cmd}"'
 
+rule download_cytobands:
+    """Downloads cytoband data for plotting & (todo) hgver-specific centromere filtering"""
+    input:
+        HTTP.remote("http://hgdownload.cse.ucsc.edu/goldenpath/hg19/database/cytoBand.txt.gz")
```

Comment:
couldn't this also replace the 'hg19' with:
"http://hgdownload.cse.ucsc.edu/goldenpath/{hgver}/database/cytoBand.txt.gz".format(hgver=config["genome_name"])

Reply:
I even included the relevant functionality in the output, and not the input. Will fix.

Reply:
Resolved in 12bee71